src/Core/src/Handlers/Image/ImageHandler.Windows.cs

```diff
@@ -55,6 +57,38 @@ protected override void RemoveContainer()
 			UpdateValue(nameof(IView.Width));
 		}
 
+		/// <inheritdoc/>
+		public override Graphics.Size GetDesiredSize(double widthConstraint, double heightConstraint)
```

Comment:
A big part of the images UITests are failing because the size of the image is different:
<img width="362" height="458" alt="image" src="https://github.com/user-attachments/assets/6ce0b488-8dab-4203-b5d9-6374ca3d04bd" />
Could you take a look? If the new result is the expected one, could you update the reference snapshots?